awx/ui_next/src/screens/Template/WorkflowJobTemplate.jsx

```diff
@@ -47,19 +49,21 @@ class WorkflowJobTemplate extends Component {
         const {
           data: { webhook_key },
         } = await WorkflowJobTemplatesAPI.readWebhookKey(id);
-        this.setState({ webHookKey: webhook_key });
+        this.setState({ webhook_key });
```

Comment:
I'm not sure if this matters or not - but I expected this variable to be initialized up in the constructor with all the other state variables.

Reply:
I don't know that it matters either, but I can add it with a null value.